src/networks.json

```diff
@@ -18,6 +18,24 @@
       "symbol": "WETH"
     }
   },
+  "42161": {
+    "name": "Arbitrum",
+     "rpc": "https://arb1.arbitrum.io/rpc",
+     "chainId":42161 ,
+     "explorer": "https://arbiscan.io/",
+     "color": "#e7eaf3",
+     "multicall": "0xcA11bde05977b3631167028862bE2a173976CA11",
```

Comment:
Did you check Arbitrum work, maybe on a testnet? Because you specified 3rd version of the Multicall contract. But our max version is 2nd. This is just to be sure it will be fine and we won't need to fix anything )

@davidmansion you forget to add a network logo. The second step here:
https://github.com/noxonsu/unifactory/blob/main/docs/addNetwork.md

Reply:
@NotEternal please specify required multical (and other contracts) version and add a link to github sources (v2 in case of multical) to https://github.com/noxonsu/unifactory/edit/main/docs/addNetwork.md

Reply:
@NotEternal Arbitrum doesn't seem to be listed on V2.
There is a logo for Arbitrum already in the files.

Reply:
Sorry for delays. And for the logo, I didn't see it's already there.
@davidmansion there did you get a Multicall V3 address? Just from the Arbitrum explorer?

Reply:
An active contract repository:
https://github.com/mds1/multicall

As I can see it's compatible with old versions:
![image](https://user-images.githubusercontent.com/61930014/226928641-e075e5e9-7843-4832-83af-bd277c4a9fbd.png)

Reply:
So the only question - did you start and check application work?

Reply:
So the only question then - did you check application work?

Reply:
> @NotEternal I got the MulticallV3 address from the Multicall GitHub repository. https://github.com/mds1/multicall

Reply:
> > @NotEternal I got the MulticallV3 address from the Multicall GitHub repository. https://github.com/mds1/multicall

No it doessn't work as its not merged yet.

Reply:
> So the only question then - did you check application work?

I found L2 Multicall contract address now.
0x842eC2c7D803033Edf55E478F461FC547Bc54EB2
Would I still need to test? As I am struggling to see how to start Unifactory locally.
https://arbiscan.io/address/0x842eC2c7D803033Edf55E478F461FC547Bc54EB2